### ISSUE TITLE
Deprecated features appear in burndown

### ISSUE THREAD

**Issue by Elchi3** [ddbeck/web-features-burndown-tools#4]
Is it intentional that deprecated features appear in the burndown? 

The VR API (api.VRDisplay and friends) for example is listed as "not cited by web-features" but my understanding was that deprecated shouldn't be in web-features at all?

**Comment by ddbeck** [comment]
I think it's more of an open question. See https://github.com/web-platform-dx/web-features/issues/311.

I think it makes sense to do something special for deprecated keys. Maybe add a column for deprecated status, so we can put some conditional formatting on those rows? Would you be open to that? Because I don't think we should act as if the keys don't exist, at least not until [the issue](https://github.com/web-platform-dx/web-features/issues/311) is settled.

**Comment by Elchi3** [comment]
Yeah, a column for deprecated status would be good. I guess a question for me is also whether or not these should be part of the general burndown that we are currently working on. If I see it correctly, the current count of deprecated keys is 1071. So a somewhat substantial amount. 

The current feature statistics say there are 7767 keys to go, or 6696 if we decide to not create features for deprecated keys.
For example, for a recent feature that I worked on, MathML, there are 39 deprecated keys, that weren't included, but appear as unmapped in the weekly feature statistics.

**Comment by ddbeck** [comment]
I've opened https://github.com/ddbeck/web-features-burndown-tools/pull/11 and updated the actual Google Sheets burndown to show what it would look like with deprecated and standard track columns appended.

I could also extend the scripts for generating stats to show different coverage % (all keys in scope and all standard, non-deprecated keys in scope).

Over on web-features, it might be nice to just throw when we hit deprecated or non-standard keys, instead of treating them as having some sort of meaning, until https://github.com/web-platform-dx/web-features/issues/311 gets sorted.
